src/Microsoft.DotNet.GitHub.Authentication/GitHubTokenProvider.cs

```diff
@@ -84,9 +84,9 @@ private bool TryGetCachedToken(long installationId, out AccessToken cachedToken)
 
         AccessToken token = _tokenCache[installationId];
 
-        // If the cached token will expire in less than 30 minutes we won't use it and let GetTokenForInstallationAsync generate a new one
+        // If the cached token will expire in less than 50 minutes we won't use it and let GetTokenForInstallationAsync generate a new one
```

Comment:
Maybe its worth mentioning the github issue in this comment as an explanation for why the time is set to such a value? In case someone has to alter this code in the future or runs into something similar

Reply:
this _lenthens_ the token lifetime to 50 minutes. I thought the intent was to shorten it to 10 minutes❓